services/storage/src/simcore_service_storage/modules/celery/worker.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from collections.abc import Callable
+
+from celery import Celery
+from celery.contrib.abortable import AbortableTask
+from models_library.progress_bar import ProgressReport
+
+from .models import TaskID
+
+_logger = logging.getLogger(__name__)
+
+
+class CeleryTaskQueueWorker:
+    def __init__(self, celery_app: Celery) -> None:
```

Comment:
The name is horrible. I really don't get what this is used for of what. Could you maybe rename it so that i makes more sense. The `Queue` part is not saying anything.
Why did we not keep tha `ClientInterface` and `WorkerInterface` patterns?

Reply:
IMHO: ClientInterface and WorkerInterface are too generic names.